recipes/bioconductor-genomicranges/meta.yaml

```diff
@@ -54,4 +54,5 @@ extra:
     name: bioconductor-genomicranges
     path: recipes/bioconductor-genomicranges
     version: 1.32.7
-
+  additional-platforms:
```

Comment:
It is not a good idea to modify several recipes in one PR!
Apart from that there are more issues here:
1) `build > number` is not bumped
2) all `bioconductor-**` recipes cannot be built on Linux ARM64 due to issue with `umask`. See https://github.com/bioconda/bioconda-recipes/pull/46177